elytron/src/main/java/org/wildfly/extension/elytron/RealmParser.java

```diff
@@ -193,7 +216,24 @@
             .addChild(keyStoreRealmParser)
             .addChild(propertiesRealmParser_14_0)
             .addChild(ldapRealmParser)
-            .addChild(filesystemRealmParser_14_0)
+            .addChild(filesystemRealmParser_15_0)
+            .addChild(tokenRealmParser)
+            .addChild(cachingRealmParser)
+            .addChild(distributedRealmParser)
+            .addChild(failoverRealmParser)
+            .addChild(jaasRealmParser)
+            .build();
+
+    final PersistentResourceXMLDescription realmParser_15_1 = decorator(ElytronDescriptionConstants.SECURITY_REALMS)
```

Comment:
We need to make sure that `realmParser_15_1` is copy/pasted from `realmParser_15_0` and only the `filesystemRealmParser` line should be updated.

Currently, `realmParser_15_1` is missing a couple children from 15_0.